File: MPI/mpicheck/tests/functions/4.0/MPI_Pack_external_c.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    const char var_0[2];
    const void *var_1;
    MPI_Count var_2;
    MPI_Datatype var_3;
    void *var_4;
    MPI_Count var_5;
    MPI_Count *var_6;
    int ret;
    /* calls */
    ret = MPI_Pack_external_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6);
    ret = PMPI_Pack_external_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6);
    return 0;
}

File: MPI/mpicheck/tests/functions/4.0/MPI_Pack_external_c.f90

        
        program main
        use mpi
        
        CHARACTER*(10) var_0
       TYPE(INTEGER) var_1(10)
       INTEGER(KIND=MPI_COUNT_KIND) var_2
       INTEGER var_3
       TYPE(INTEGER) var_4(10)
       INTEGER(KIND=MPI_COUNT_KIND) var_5
       INTEGER(KIND=MPI_COUNT_KIND) var_6
       INTEGER var_7
        call mpi_pack_external_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7)
       call pmpi_pack_external_c(var_0, var_1, var_2, var_3, var_4, var_5, var_6, var_7)
        end program main
    